cmake/CudaRdcUtils.cmake

```diff
@@ -1,5 +1,6 @@
-#------------------------------- -*- cmake -*- -------------------------------#
-# Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+#----------------------------------*-CMake-*----------------------------------#
+# Copyright 2020-2024 UT-Battelle, LLC, and other Celeritas developers.
+# See the top-level COPYRIGHT file for details.
```

Comment:
Please update the "source" RdcUtils with the updated template and revert this

Reply:
Somehow the version in VecGeom was uploaded with white space changes that were not propagated to Celeritas nor the 'source' (CudaRDCExercise) and in the diff length I missed that actual change :(

Reply:
Reverting would work for the version in Celeritas but not for the other 2 copies which might not have have the right copyright nor license files ...

Reply:
I uploaded an update proposal for the copyright lines.

Reply:
Updated copyright looks good to me, thanks.